packages/vats/test/test-network.js

```diff
@@ -185,11 +185,19 @@ test('network - ibc', async t => {
       portID: 'port-1',
       channelID: 'channel-1',
       counterparty: { port_id: 'port-98', channel_id: 'channel-22' },
-      counterpartyVersion: 'bar',
+      counterpartyVersion: 'bar-negotiated',
       connectionHops: ['connection-11'],
     });
 
     const c = await when(cP);
+    const remoteAddress = c.getRemoteAddress();
+    const localAddress = c.getLocalAddress();
+    t.is(
+      remoteAddress,
+      '/ibc-hop/connection-11/ibc-port/port-98/unordered/bar-negotiated/ibc-channel/channel-22',
+    );
+    t.is(localAddress, '/ibc-port/port-1/ibc-channel/channel-1');
```

Comment:
Before:
```
remoteAddress: '/ibc-hop/connection-11/ibc-port/port-98/unordered/bar'
localAddress: '/ibc-port/port-1'
```

Reply:
I would even go further, and make it so that `localAddress` is `/ibc-port/port-1/unordered/bar-negotiated/ibc-channel/channel-1`.  I think that's how (at least) `vat-ibc.js` behaved at one time.

Reply:
[1137317](https://github.com/Agoric/agoric-sdk/pull/9186/commits/113731740741d8a61bbe1e8cca0b9bfc959043da)